ch7/exercise7-6/tempflag.go

```diff
@@ -2,16 +2,15 @@
 package main
 
 import (
-	"flag"
 	"fmt"
 
 	"./tempconv"
+	"gopkg.in/alecthomas/kingpin.v2"
 )
 
-//!+
 var temp = tempconv.CelsiusFlag("temp", 20.0, "the temperature")
 
 func main() {
-	flag.Parse()
+	kingpin.Parse()
```

Comment:
I'd encourage you to try out the "no global state" app syntax, as that is what teleport uses: https://github.com/alecthomas/kingpin#complex-example